### PR TITLE
feat: upgrade step to clean TS api URL in registry to keep only base URL

### PR DESCRIPTION
**Pull request by remdub**
CITI-15

## Summary by CodeRabbit

* **New Features**
  * Automatic upgrade to normalize the TS API URL in configuration so only the base URL is retained.

* **Tests**
  * Added tests covering TS API URL validation, normalization, and edge cases.

* **Chores**
  * Bumped policy profile version to 1041.

* **Documentation**
  * Updated changelog/unreleased entry to reflect the upgrade and removed the placeholder note.

### CODE REVIEW

**Comment by remdub** [comment]
@coderabbitai review

File: src/imio/smartweb/policy/utils.py [hunk 1]
```diff
@@ -284,3 +285,15 @@ def get_accessibility_html_content():
...
+def get_ts_api_base_url() -> str | None:
+    """Get the base url of the ts api."""
+    from imio.smartweb.core.utils import get_value_from_registry
+    from imio.smartweb.core.utils import is_valid_url
+
+    url_ts = get_value_from_registry("smartweb.url_ts")
+    if is_valid_url(url_ts):
+        parsed_url = urlparse(url_ts)
+        return f"{parsed_url.scheme}://{parsed_url.netloc}/"
+    return None
```
[the remark this answers is not in the document]
Comment (remdub) [reply src/imio/smartweb/policy/utils.py:299]: there won't be any credentials in this variable
